recipes/sssp/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+mkdir -p ${PREFIX}/share/sssp
+cp -r efficiency ${PREFIX}/share/sssp
+cp -r precision ${PREFIX}/share/sssp
```

Comment:
I am not sure if these directories will work as expected in a generic noarch pkg.

Reply:
So your suggestion is to skip the windows build? The DFT simulation codes which use these pseudo potentials are only available for Linux so I am even fine with restricting it to Linux.

Reply:
@ocefpaf Can you take another look?

Reply:
To be honest I don't know! We can merge this as noarch, test on Windows, and mark as broken if they do not work.